### ISSUE TITLE
Moodle 4.0 "Add an activity or resource" icons are not visible in high- nor low-contrast

### ISSUE THREAD

**Issue by JosephKennedy4** [brickfield/moodle-block_accessibility#143]
Many of the icons in the "Add an activity or resource" pop-up are not visible when using Accessibility block blue- or black-background views in Moodle 4.0.2
